test-suite/src/test/groovy/io/micronaut/docs/http/client/proxy/ProxyRequestSpec.groovy

```diff
@@ -102,7 +102,7 @@ class ProxyRequestSpec extends Specification {
         response.body().text == 'Hello John XXX'
 
         when:"A POST request is sent with an invalid content type"
-        response = client.exchange(HttpRequest.POST("/proxy/post/json", new Message(text: "John")).contentType(MediaType.TEXT_PLAIN), Message).blockFirst()
```

Comment:
breaking change, can't write arbitrary type (Book) as text/plain anymore

Reply:
can you explain why?

Reply:
TextPlainCodec used to just accept any object and call toString on it. TextPlainHandler only works with String, so it gives an error that there is no handler for text/plain for this type. i think it makes more sense.